README.md

```diff
@@ -18,27 +34,56 @@ The community calendar contains all the upcoming OpenFeature meetings and events
   Calendar](https://calendar.google.com/calendar/u/0?cid=MHVhN2kxaGl2NWRoMThiMjd0b2FoNjM2NDRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
 - [iCal](https://calendar.google.com/calendar/ical/0ua7i1hiv5dh18b27toah63644%40group.calendar.google.com/public/basic.ics)
 
-Notes and recordings from previous meetings can be found below:
 
-- [Meeting notes](https://docs.google.com/document/d/1pp6t2giTcdEdVAri_2B1Z6Mv8mHhvtZT1AmkPV9K7xQ/edit?usp=sharing)
-- [Recordings](https://www.youtube.com/channel/UCXSFt-dT2HORGXz1-ksxtxw)
+### Roadmap
 
-## Slack
+The project roadmap is available [here](https://github.com/orgs/open-feature/projects/1).
+New initiatives require an accepted [OpenFeature Enhancement Proposal (OFEP)](https://github.com/open-feature/ofep). A community decision will then be made as documented in [Decision Making](./governance-charter.md#decision-making).
 
-You can find us in the `#openfeature` channel on the CNCF Slack Workspace. You can join the CNCF Slack Workspace by registering [here](https://slack.cncf.io).
+### Mailing Lists 
+
+TODO
```

Comment:
Please create an issue for this and remove the TODO. We can add this section back once we have a mailing list to use.

Reply:
removed